back/config/schemas/settings.schema.json.erb

```diff
@@ -1304,6 +1304,18 @@
         }
       },
 
+      "customisable_homepage_banner": {
+        "type": "object",
+        "title": "Customisable homepage banner",
+        "description": "Enables customization of various settings for the homepage banner, including layout options, overlay colors, and more, beyond just the banner text.",
```

Comment:
@EdwinKato I can still set the overlay colors (and opacity) while allowed is set to false. Shall we just not show these when allowed is false? It makes sense to have toggle turned on and locked as is the case, but then not show those other configuration options.

<img width="389" alt="image" src="https://github.com/CitizenLabDotCo/citizenlab/assets/16427929/9244d4a2-b7b3-4a70-aa1c-e2b24f4c71f3">